pkgs/http_profile/lib/src/http_profile_response_data.dart

```diff
@@ -176,11 +184,12 @@ final class HttpProfileResponseData {
   ///
   /// [endTime] is the time when the response was fully received. It defaults
   /// to the current time.
-  void close([DateTime? endTime]) {
+  Future<void> close([DateTime? endTime]) async {
```

Comment:
I wonder if we should but a note here - and in the http_profile_request_data equivalent - that it is safe to unawait this Future? Probably we don't need to say that.